Playbooks/AS-Clear-Okta-Network-Zone-List/azuredeploy.json

```diff
@@ -0,0 +1,395 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "AS-Clear-Okta-Network-Zone-List", 
+        "description": "This playbook is intended to be run on a schedule. It will clear out all but one of the IPs from an Okta Network Zone list (leaving at least one entry is required by API). This will prevent the list from reaching capacity and preventing newer entries from being added.",
+        "prerequisites": "1. An Okta admin account is needed, along with an API token. 2. A Microsoft Azure key vault containing the Okta API token must also be set up. Support for the set up and configuration of each of these items can be found here: https://github.com/Accelerynt-Security/AS-Clear-Okta-Network-Zone-List",
+        "postDeployment": "1. Access to the Azure Key Vault secret must be granted to the playbook."
```

Comment:
Please add , at the end. 
String not closed properly. ' , '  is expected at the end

Reply:
@v-prasadboke this is fixed